tests/test_warc_metadata_sidecar.py

```diff
@@ -0,0 +1,65 @@
+import io
+import os
+from logging import INFO
+from unittest.mock import patch
+
+from warcio.archiveiterator import ArchiveIterator
+from warcio.recordloader import ArcWarcRecord
+
+import warc_metadata_sidecar as sidecar
+
+
+class Test_Warc_Metadata_Sidecar:
+    def get_record(self):
+        test_file = os.path.join(os.path.dirname(__file__), 'warc.warc')
+        with open(test_file, 'rb') as stream:
+            for record in ArchiveIterator(stream):
+                url = record.rec_headers.get_header('WARC-Target-URI')
+                rawPayload = io.BytesIO(record.raw_stream.read())
+                if record.http_headers:
+                    recordCopy = ArcWarcRecord(
+                        record.format, record.rec_type,
+                        record.rec_headers, rawPayload,
+                        record.http_headers, record.content_type,
+                        record.length
+                        )
+                    decodedPayload = recordCopy.content_stream().read()
+                    rawPayload.seek(0)
+                else:
+                    decodedPayload = rawPayload
+            return(url, rawPayload, decodedPayload)
+
+    @patch('warc_metadata_sidecar.WARCWriter')
+    def test_metadata_sidecar(self, mock_warcwriter, caplog, tmpdir):
+        caplog.set_level(INFO)
+        sidecar.metadata_sidecar(str(tmpdir), 'tests/test_warc.warc')
```

Comment:
Perhaps use `os.path.join(os.path.dirname(__file__)` in place of hard coding the path to the tests dir in 'tests/test_warc.warc'. You could set `os.path.join(os.path.dirname(__file__)` as a constant since you use it elsewhere in the file.